### DISCUSSION TITLE
A question about Attributes

### DISCUSSION THREAD

**Discussion by Psi25** [bnpr/Malt#373] [Q&A] [answered]
Does Malt support attributes? Let's say I have a attribute(something like from geonodes) that contains UV data and I want to make use of that in Malt. Is it possible?

**Comment by dibli-goost** [comment, marked as answer]
I think with the case of UVs, you could just write the data to a new UV Map rather than to an "attribute"?

With other cases, like normals, you need to use Color Attributes (vertex colors). That's what the Vertex Color Overrides are for in the object data properties.

See this thread for more info: #142
